src/os/memstore/MemStore.cc

```diff
@@ -795,6 +795,12 @@ void MemStore::_do_transaction(Transaction& t)
       }
       break;
 
+    case Transaction::OP_MERGE_DELETE:
+      {
+       assert(0 == "not implemented");
```

Comment:
This does need to be implemented.  We'll want to be able to test ec overwrites in memory...